src/main/webapp/app/teams/teams-skills.scss

```
@@ -24,6 +24,24 @@
     }
 }
 
+.score-popover{
+    background-color: #666666;
```

This color is available as a SCSS variable:
```suggestion
    background-color: $dojo-text-primary;
```